gbpservice/nfp/configurator/lib/lb_constants.py

```diff
@@ -0,0 +1,50 @@
+# DRIVERS_DIR = '/usr/lib/python2.7/dist-packages/gbpservice/nfp/'\
+#              'configurator/drivers/loadbalancer/'
+DRIVERS_DIR = '/opt/stack/gbp/gbpservice/nfp/configurator/drivers/loadbalancer/'
```

Comment:
We need to fix this driver path specification issue. What if the path changes from setup to setup? How will we take the input.